ui/app/appointments/views/manage/list/defaultListPrint.html

```diff
@@ -59,7 +59,9 @@ <h5 ng-if="!searchedPatient">{{::'APPOINTMENT_DATE' | translate}} {{::startDate
                     <th class="table-mid-width" ng-if="enableServiceTypes">{{::'APPOINTMENT_SERVICE_TYPE_FULL' |translate}}</th>
                     <th class="table-mid-width">{{::'APPOINTMENT_SERVICE_LOCATION_KEY' |translate}}</th>
                     <th>{{::'APPOINTMENT_STATUS' |translate}}</th>
-                    <!--<th class="notes">{{::'APPOINTMENT_COMMENTS' |translate}}</th>-->
+                    <!--<th>{{::'APPOINTMENT_ADDITIONAL_INFO' |translate}}</th>
+                    <th ng-repeat-start="columnValue in additionalInfoColumns" class="table-mid-width" ng-repeat-end>'APPOINTMENT_ADDITIONAL_INFO_COLUMN_'+columnValue.toUpperCase() \ translate</th>
+                    <th class="notes">{{::'APPOINTMENT_COMMENTS' |translate}}</th>-->
```

Comment:
Remove commented lines

Reply:
These commented statements were purposely retained because, in case, any implementor wants to have those columns, he/she can just uncomment the same and use.